larq/conftest.py

```diff
@@ -73,3 +73,8 @@ def quantized(request):
     """pytest fixture for running test quantized and non-quantized"""
     with lq_context.quantized_scope(request.param):
         yield request.param
+
+
+@pytest.fixture(params=["channels_last", "channels_first"])
+def data_format(request):
+    return request.param
```

Comment:
Could you explain what this does? From [the pytest docs](https://docs.pytest.org/en/stable/fixture.html#fixtures-can-introspect-the-requesting-test-context) I understand that `request.param` refers to some property of the calling function?

Reply:
When using `data_format` as an argument to a testing funciton pytest will run the function twice, once with `request.param=="channels_last"` and once with `request.param=="channels_first"`. See [here](https://github.com/larq/larq/pull/506/files#diff-fd74c9d55bedda7f687491b4acd9e468L223-R223) for a usage example.